src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,21 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+            UserService userService = new UserService();
+            User user = userService.findByEmail(email);
+            return user != null && email.equals(user.getEmail()) && password.equals(user.getPassword());
```

Comment:
```suggestion
            return user != null && password.equals(user.getPassword());
```
you already found user by email, so they are equal